profiles/mojoshader_profile_glsl.c

```diff
@@ -1776,7 +1783,11 @@ static void glsl_texld(Context *ctx, const int texldd)
                 else  // texld/texldb
                 {
                     funcname = "texture3D";
-                    make_GLSL_srcarg_string_vec3(ctx, 0, src0, sizeof (src0));
+                    // !!! FIXME: I have not tested this.
```

Comment:
Looking further down this is correct; the src0 operand is being treated the same way, it just has a `z` for 3D. Not much to see here...